new/LIT7058Sababata.xml

```diff
@@ -0,0 +1,79 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A <term key="Qene">qǝne</term>of possibly <term key="Mawaddes">mawaddǝs</term> type.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-09-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7058Sababata" passive="BNFabb145"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" xml:lang="gez">
+                    <note>The text is taken from <ref type="mss" corresp="BNFabb154"/>, fol. 56vc
+                         (division into verses is indicated with the sign ።.</note>
+                    <ab>                                                  
+                        <l n="1">ጸበበተነ፡ ምድርከ፡ በእለተ፡ እሑድ፡ እስመ፡ ፈጠርካ፡ ስፍ<supplied reason="undefined" resp="DN">ሐ</supplied><gap reason="illegible"/>፡ እግዚአብሔር፡ ፌማ።</l> 
+                        <l n="2"><supplied reason="undefined" resp="DN">እስ</supplied>መ፡ ሶቤሃ፡ ተሰምዓ፡ መዓተ፡ ኃይልከ<gap reason="illegible"/>ማ።</l>
+                        <l n="3">ወእንበለ፡ ዓቅም፡ ተነዝኃ፡ ፍጽማ።</l>  
+                        <l n="4">ደመ፡ አንክሮ፡ ምክረ፡ <supplied reason="undefined" resp="DN">ማ</supplied>ስቴማ።</l>
+                        <l n="5">ሐዊረ፡ ፍኖትኒ፡ በአ<gap reason="illegible"/>፡ ዕበይ፡ ወግርማ።</l>
```

Comment:
One possible solution for the gap could be አሚን፡, but this is rather hypothetic.
I suggest to add unit="char" quantity="2", to indicate, that it is a rather small gap with space for 2 characters in maximum.
I think, it is not possible to mark up the reason of a gap, that is not readable in the images because of the neat binding. This could be useful to add in a note. Some users may have access to the manuscript and can update the reading in future.

Reply:
Amin would be fine, mi is unlikely after ba-a... (it could be also za-a..)